operations/mimir/config.libsonnet

```diff
@@ -535,6 +535,19 @@
     //    },
     distributor_instance_limits: null,
 
+    // Controls whether the TSDB Head early compaction is enabled in the ingester.
+    ingester_tsdb_head_early_compaction_enabled: false,
+    ingester_tsdb_head_early_compaction_reduction_percentage: 15,
```

Comment:
I suppose we can also change the default of the `early-head-compaction-min-in-memory-series` too to 2M (or to match the instance limit, which may be more difficult)

Reply:
See: https://github.com/grafana/mimir/pull/6186

Reply:
>I suppose we can also change the default of the early-head-compaction-min-in-memory-series too to 2M (or to match the instance limit, which may be more difficult)

Setting the default to 2M means it's enabled by default but I think we shouldn't do it. The problem of enabling it by default is that the “min in-memory series threshold” may be completely off your actual ingesters size. That’s why I believe it will be hard to enable it by default In Mimir, but we may enable it by default in jsonnet / helm.